x/concentrated-liquidity/spread_rewards_test.go

```diff
@@ -1359,6 +1360,27 @@ func (s *KeeperTestSuite) TestUpdatePosValueToInitValuePlusGrowthOutside() {
 	}
 }
 
+func NewRandomizedPositions() Positions {
+	rand.Seed(time.Now().UnixNano())
+
+	numSwaps := 2 + rand.Intn(99) // Random number between 2 and 100
+	numAccounts := 2 + rand.Intn(99)
+
+	numFullRange := rand.Intn(numAccounts + 1) // Random number between 0 and numAccounts inclusive
```

Comment:
It might be helpful to reference how we bound random inputs in the simulator. e.g. in this specific case, we probably want to have _some_ cap on the number of positions so tests don't run for 10+ minutes

Tldr is you can mod the output to bound the range of the final number, similar to what's done in #5476